webpack.config.js

```diff
@@ -6,10 +6,17 @@ module.exports = {
     	publicPath: '/static/'
 	},
 	module: {
-		loaders: [{
-			test: /\.js$/,
-			exclude: /node_modules/,
-			loader: "babel-loader"
-		}]
+		loaders: [
+			{
+				test: /\.js$/,
+				exclude: /node_modules/,
+				loader: "babel-loader"
+			},
+			{ test: /\.css$/, loader: "style-loader!css-loader" },
```

Comment:
There is a error msg "Module not found: Error: Cannot resolve module 'style-loader'" so I can't run the server.